_includes/variables.html

```diff
@@ -69,7 +69,7 @@
     {% endif %}
   {% endif %}
 <!-- Doc link -->
-  {% assign doc_link = lang | append: "/doc/" %}
+  {% assign doc_link = "https://doc.qubes-os.org/" | append: page.lang | append: "/latest/index.html" %}
```

Comment:
Since `page.lang` could be `nil`, I suggest the following lines instead of this line 72:

```
{% if page.lang == nil %}
  {% assign doclang = "en" %}
{% else %}
  {% assign doclang = page.lang %}
{% endif %}
{% assign doc_link = "https://doc.qubes-os.org/" | append: doclang | append: "/latest/index.html" %}
```